src/index.ts

```diff
@@ -60,6 +63,10 @@ export = class OutputCsv {
     }
   }
 
+  private isRecord(value: unknown): value is Record<string, unknown> {
+    return typeof value === 'object' && value !== null;
```

Comment:
Don't we have lodash in this project? If so we can just use `_.isPlainObject`: https://docs-lodash.com/v4/is-plain-object/

Reply:
I agree. Used `_.isPlainObject` instead of type guarding function.